core/objects.go

```diff
@@ -228,7 +229,8 @@ func (ch Challenge) RecordsSane() bool {
 			ch.ValidationRecord[0].AddressUsed == nil || len(ch.ValidationRecord[0].AddressesResolved) == 0 {
 			return false
 		}
-	case ChallengeTypeDNS01:
+	case ChallengeTypeDNS01,
+		ChallengeTypeDNSAccount01:
```

Comment:
nit: it feels weird to have this (shorter) case broken onto two lines, while the similar (longer) case on line 65 is all one line. 
```suggestion
	case ChallengeTypeDNS01, ChallengeTypeDNSAccount01:
```